posts/2024-07-26-actuated-for-gitlab.md

```diff
@@ -0,0 +1,217 @@
+---
+title: Secure CI for GitLab with Actuated.
+description: Learn how Actuated can improve your GitLab CI.
```

Comment:
`Learn how Actuated can improve your GitLab CI.`

=>

`Learn how microVMs provide more flexibility and security for CI workloads, by removing the need for privileged containers or Docker In Docker.`